### ISSUE TITLE
index.html

### ISSUE THREAD

**Issue by sum-in-h** [sum-in-h/sum-in-h.github.io#1]
<!DOCTYPE html>
<html>
    
    <head>
        <title> Exercice #8  </title>
        <meta charset="utf-8">
        <meta name="viewport"
        content = "width=divice-width, initial-scale=1">
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
       <style>
         .bg-1{
          background-color: #1abc9c; color : #ffffff
         }
         .bg-2{
          background-color: gray; color : #ffffff
         }
         .bg-3{
          background-color: #ffffff; color : gray
         }
         .bg-4 {
          background-color: #2f2f2f;
          color: #ffffff;
         }
         .container-fluid {
          padding-top : 70px; 
          padding-bottom :70px
         }
          .navbar{
            padding-top: 15px;
            padding-bottom: 15px;
            border: 0;
            margin-bottom: 0;
            font-size: 12px;
            letter-spacing: 5px;
          }
          .navbar-nav li a:hover{
            color: #1abc9c !important ;    
          }
          body {
            font: 20px 'Montserrat', 'sans-serif';
            line-height: 1.8;
            color: #f5f6f7;
          }
          p {
            font-size: 16px;
          }
          .margin {
            margin-bottom: 45px;
          }
          
        </style>

    </head>

    
    <body>
      <nav class="navbar navbar-default">
        <div class="container">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle"
               data-toggle="collapse" data-target="#myNavbar">
               <span class="icon-bar"></span>
               <span class="icon-bar"></span>
               <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="#">Me</a>
          </div>
          <div class="collapse navbar-collapse"
           id="myNavbar">
          </div>
          <ul class="nav navbar-nav navbar-right">
            <li class="active"></li>
            <li><a href="#">WHO</a></li>
            <li><a href="#">WHAT</a></li>
            <li><a href="#">WHERE</a></li>
          </ul>
        </div>
      </nav>
        <div class="container-fluid bg-1 text-center" >
         <h3> Who am I? </h3>
         <img src="bird.jpg" alt="Bird" width="15%" class="img-circle">
         <h3> I am an Adventurer. </h3>
        </div>

        <div class="container-fluid bg-2 text-center" >
          <h3> What am I? </h3>
          <p> "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum." </p>
          <a href="#" class="btn btn-default btn-lg">
            <span class="glyphicon glyphicon-search"></span>Search
          </a>
        </div>

          <div class="container-fluid bg-3 text-center" >
            <h3> Where To Find Me? </h3>
            <div class="row">
              <div class="col-sm-4">
                <p>"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."   </p>
                <img src="bird1.jpg" width="100%">               
              </div>
              <div class="col-sm-4">
                <p>"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum. "  </p>
                <img src="bird2.jpg" width="80%"> 
              </div>
              <div class="col-sm-4">
                <p>"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."    </p>
                <img src="bird3.jpg" width="90%">
              </div>
          </div>

        </div>

        <footer class="container-fluid bg-4 text-center">
          <p>Bootstrap Theme Made By <a href="#">Hwang SuMin</a></p>
        </footer>

    </body>

</html>
